stories/__tests__/SQFormAsyncAutocomplete.stories.test.tsx

```diff
@@ -167,12 +167,14 @@ describe('SQFormAsyncAutocomplete Tests', () => {
 
   describe('Autocomplete With Validation', () => {
     it('should render as required when it is a required field', async () => {
-      render(<SQFormAsyncAutocompleteWithValidation size="auto" />);
+      const {container} = render(
+        <SQFormAsyncAutocompleteWithValidation size="auto" />
+      );
 
       await waitFor(() => {
-        const requiredText = screen.getByText(/required/i);
-        expect(requiredText).toBeVisible();
-        expect(requiredText).toHaveClass('Mui-required');
+        const helperText = container.querySelector('.MuiFormHelperText-root');
```

Comment:
All of these changes in these tests are going to be similar. Considering we're using the typography now the parent of the "text element" is what actually has the class we're looking for. So that's what we're getting here.